Assets/Scripts/CameraManager.cs

```diff
@@ -23,28 +24,54 @@ public class CameraManager : MonoBehaviour {
         offRotation = transform.rotation.eulerAngles;
         onPosition = offPosition + new Vector3(0f, -3.5f, 4.5f);
         onRotation = offRotation + new Vector3(15, 0, 0);
+        position = offPosition;
+        rotation = offRotation;
+        focus = offFocus;
     }
 
     // Update is called once per frame
     void Update() {
-        if (Input.GetKeyDown(KeyCode.Space)) {
-            gameStarted = !gameStarted;
-        }
+ 
+    }
 
-        if (!gameStarted) {
+    public void SwitchCameraPosition()
+    {
+        gameStarted = !gameStarted;
+        if (!gameStarted)
+        {
             position = offPosition;
             rotation = offRotation;
             focus = offFocus;
-        } else {
+        }
+        else
+        {
             position = onPosition;
             rotation = onRotation;
             focus = onFocus;
         }
+        StartCoroutine(MoveCameraPosition(Time.deltaTime));
+    }
 
-        transform.position = Vector3.Lerp(transform.position, position, Time.deltaTime * speedMod);
-        transform.rotation = Quaternion.Slerp(transform.rotation, Quaternion.Euler(rotation), Time.deltaTime * speedMod);
-        
-        DepthOfField depth = (DepthOfField) gameObject.GetComponent<Volume>().profile.components[0];
-        depth.focusDistance.value = Mathf.Lerp(depth.focusDistance.value, focus, Time.deltaTime * speedMod);
+    IEnumerator MoveCameraPosition(float deltaTime)
+    {
+        if (transform.position != position)
+        {
+            transform.rotation = Quaternion.Slerp(transform.rotation, Quaternion.Euler(rotation), Time.deltaTime * speedMod);
+            if ((transform.position - position).magnitude < 0.75f)
+            {
+                transform.position = Vector3.MoveTowards(transform.position, position, Time.deltaTime * speedMod * 0.75f);
+            } else
+            {
+                transform.position = Vector3.Lerp(transform.position, position, Time.deltaTime * speedMod);
+            }
+            DepthOfField depth = (DepthOfField)gameObject.GetComponent<Volume>().profile.components[0];
+            depth.focusDistance.value = Mathf.Lerp(depth.focusDistance.value, focus, Time.deltaTime * speedMod);
+            yield return new WaitForSeconds(deltaTime);
```

Comment:
time.delta time will be from last frame. So you should actually be using yield return null or yield return 0.